README.md

```diff
@@ -9,7 +9,10 @@ from tripleo-quickstart here.
 
 # Pre-requisites
 
-- CentOS 7
+- CentOS 7.5 or greater (installed from 7.4 or newer)
+- file system that supports the following:
+    - Native Overlay Diff: "true"
```

Comment:
perhaps you could explain how to check if it's true?

Reply:
Added troubleshooting section